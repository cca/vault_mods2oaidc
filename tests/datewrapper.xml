<?xml version="1.0" encoding="UTF-8"?>
<xml>
  <mods>
    <origininfo>
      <dateType>dateCreated</dateType>
      <dateCreatedWrapper>
        <dateCreated keyDate="yes"/>
        <pointStart>2016-09-14</pointStart>
        <pointEnd>2016-12-07</pointEnd>
      </dateCreatedWrapper>
    </origininfo>
  </mods>
</xml>
