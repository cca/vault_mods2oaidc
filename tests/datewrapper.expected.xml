<?xml version="1.0"?>
<oai_dc:dc xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:identifier>https://vault.cca.edu/items//0/</dc:identifier>
  <dc:date>2016-09-14</dc:date>
</oai_dc:dc>
